deploy/controller_config.yaml

```diff
@@ -2,9 +2,10 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: che-workspace-crd-controller
+  namespace: operators
 data:
-  ingress.global.domain: 192.168.39.101.nip.io
-  plugin.registry: http://che-plugin-registry-crd-poc.192.168.39.101.nip.io/v3
+  ingress.global.domain: 192.168.99.100.nip.io
```

Comment:
Looks like an accidental change?

Reply:
Probably a change made to have it running locally :-)

Reply:
> Probably a change made to have it running locally :-)

Exactly. And `192.168.39.101` seems to be some generated value, I assume that kvm was used as VMDriver, while `192.168.99.100` should work for every in most cases when VirtualBox VMDriver is used.
So, I would leave `192.168.99.100` but I'm OK with reverting these changes as well.

P.S. I plan to add a command to README that would replace this value with real `minikube ip`

Reply:
I'm +1 for `192.168.99.100` if we're including an ip address here, though I use kvm and it doesn't allow setting an address.